plugins/devtools-backend/package.json

```diff
@@ -7,7 +7,8 @@
     "pluginPackages": [
       "@backstage/plugin-devtools",
       "@backstage/plugin-devtools-backend",
-      "@backstage/plugin-devtools-common"
+      "@backstage/plugin-devtools-common",
+      "@backstage/plugin-devtools-react"
```

Comment:
Pretty sure this is a mistake, there's no other backend changes and brining in a react package here seems very odd!

Reply:
See comment above